content/2019-07-04_vyvijime-rozhrani.md

```diff
@@ -0,0 +1,50 @@
+Title: Vyvíjíme rozhraní
+Date: 2019-07-04 16:36:00
+Modified: 2019-07-04 16:36:00
+Author: Karolina Surma
+
+
+Naše srazy postupně získaly stálý rámec, když si řekneme: 
+- co nového se udalo na projektu za poslední období, 
+- vysvětlíme případné nesrovnalosti v otevřených PR, 
+- a naplánujeme, na čem budeme pracovat dál.
+
+Jelikož už méně přednáškujeme a jsme víc soustředěné na konkrétní problémy, vyplácí se nám práce s pracovním dokumentem s agendou, kam vkládáme vše, co nás během týdne napadne a co chceme probrat s expertem.
```

Comment:
Tady váhám - "přednáškujeme" není existující slovo, ale líbí se mi :smile: 
Pokud bys to chtěla nějak víc česky, muselo by se to asi nějak opsat. Něco jako "Jelikož už jsou naše srazy méně o přednáškách, ale jsme víc..."

Reply:
Viděla jsem, že mi to slovo svítí červeně, ale pořád se mi líbí. ;) I v google se objevuje, tak to vypadá, že vyplňuje díru v systému